docker/standalone.php

```diff
@@ -502,24 +493,27 @@ function initUI() {
 			
         }
 
-        @media (max-width: 614px) {
-            .testArea,
-            .testArea2 {
-                width: 100% !important;
-                padding: 12px !important;
-                border-radius: 8px !important;
-                gap: 6px !important;
-            }
-
-            .title-class {
-                font-size: 24px !important;
-            }
+        @media (max-width: 750px) {
+			.container .testArea,
+			.container .testArea2 {
+				max-width: 150px;
```

Comment:
We discussed this. No max-width for mobile

Reply:
fixing this right away